### PR TITLE
Document replay extensions for DevTools Recorder

### PR DESCRIPTION
**Pull request by OrKoN**
Changes proposed in this pull request:

- Documents the latest additions to the Recorder extension API

### CODE REVIEW

**Comment by OrKoN** [comment]
@jecfish PTAL https://pr-5135-static-dot-dcc-staging.uc.r.appspot.com/docs/extensions/reference/devtools_recorder

**Review by jecfish — CHANGES_REQUESTED** [review]

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Approve the content. Request a tech writer review, as sofia is OOO. @AaronForinton

File: site/en/docs/extensions/reference/devtools_recorder/index.md [hunk 1]
```diff
@@ -44,7 +46,54 @@ chrome.devtools.recorder.registerRecorderExtensionPlugin(
 );
 ```
 
+## Customizing the replay button
+
```
[the remark this answers is not in the document]
Comment (OrKoN) [reply site/en/docs/extensions/reference/devtools_recorder/index.md:50]: Thanks a bunch!

**Review by AaronForinton — APPROVED** [review]

**Comment by AaronForinton** [comment]
@jecfish Approved and ready to merge.

File: site/en/docs/extensions/reference/devtools_recorder/index.md [hunk 2]
```diff
@@ -23,7 +25,7 @@ The [`mediaType`][4] parameter allows the extension to specify the kind of outpu
 `stringify` and `stringifyStep` functions. For example, `application/javascript` if the result is a JavaScript
 program.
 
-## Examples
+## Export plugin example
```
Comment (jecfish) [inline site/en/docs/extensions/reference/devtools_recorder/index.md:28]:
```suggestion
## Export plugin example 
```

**Pull request merged**